play/src/components/openanswercard/OpenAnswerCard.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import { Typography, Box } from '@mui/material';
+import { isNullOrUndefined, ITeamAnswerContent, INormAnswer, GameSessionState } from '@righton/networking';
+import { evaluate } from 'mathjs';
+import * as DOMPurify from 'dompurify';
+import ReactQuill from 'react-quill';
+import katex from "katex";
+import './ReactQuill.css';
+import "katex/dist/katex.min.css";
+import { handleNormalizeAnswers } from '../../lib/HelperFunctions';
+import { AnswerType, StorageKeyAnswer, LocalModel } from '../../lib/PlayModels';
+import BodyCardStyled from '../../lib/styledcomponents/BodyCardStyled';
+import BodyCardContainerStyled from '../../lib/styledcomponents/BodyCardContainerStyled';
+import ButtonSubmitAnswer from '../ButtonSubmitAnswer';
+
+window.katex = katex;
+
+interface OpenAnswerCardProps {
+  answerContent: ITeamAnswerContent;
+  isSubmitted: boolean;
+  currentState: GameSessionState;
+  currentQuestionIndex: number;
+  handleSubmitAnswer: (result: ITeamAnswerContent) => void;
+}
+
+export default function OpenAnswerCard({
+  answerContent,
+  isSubmitted,
+  currentState,
+  currentQuestionIndex,
+  handleSubmitAnswer,
+}: OpenAnswerCardProps) {
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const modules = {
+    toolbar: [
+      ['formula']
+    ],
+  };
+  const formats = [
+    'formula'
+  ];
+  // these two functions isolate the quill data structure (delta) from the rest of the app
+  // this allows for the use of a different editor in the future by just adjusting the parsing in these functions
+  const insertQuillDelta = (inputAnswer: ITeamAnswerContent) => {
+    return inputAnswer.delta ?? [];
+  };
+
+  const [editorContents, setEditorContents] = useState<any>(() => insertQuillDelta(answerContent));
+
+  // ReactQuill onChange expects four parameters
```

Comment:
approach here is minimize any work that occurs on change. we store the quill delta (the storage object quill uses for content) wholesale into localstorage and then reload this into the quill on refresh. We don't do any normalization, etc, on the data until the user submits the answer